docs/README.md

```diff
@@ -1,5 +0,0 @@
-## Building the docs locally (continuously)
-```
-cd giskard
-pdm run watch-doc
```

Comment:
why remove it? let's just correct it `pdm watch-doc`

Reply:
it's inside the doc folder source, and so it's watched by sphinx